controllers/drplacementcontrolvolsync.go

```diff
@@ -330,21 +329,20 @@ func (d *DRPCInstance) ResetVolSyncRDOnPrimary(clusterName string) error {
 		return nil
 	}
 
-	vrgMWName := d.mwu.BuildManifestWorkName(rmnutil.MWTypeVRG)
-	d.log.Info(fmt.Sprintf("Resetting RD VRG ownedby MW %s for cluster %s", vrgMWName, clusterName))
-
-	mw, err := d.mwu.FindManifestWork(vrgMWName, clusterName)
+	mw, err := d.mwu.FindManifestWork(rmnutil.MWTypeVRG, clusterName)
```

Comment:
@ELENAGER we didn't catch this in the code review.  It should be `FindManifestWorkByType`...

Reply:
@BenamarMk, right! Great catch, thank you!